### ISSUE TITLE
Build on MacOS High Sierra

### ISSUE THREAD

**Issue by Kathryn-Drake** [ludvigak/FINUFFT.jl#10]
This is a great tool, and I look forward to using it.
I am trying to build FINUFFT in Julia. I am running High Sierra, I have Julia v1.1.1, and I have both gcc 8 and curl installed. I've followed the instructions in the documentation to add FINUFFT and test FINUFFT. However, when I try to test FINUFFT, I get the error 
"ERROR: LoadError: FINUFFT is not properly installed. Please build it first."
Indeed the file "deps.jl" does not exist in the deps folder. 
Is there a way I can try to manually build this?

**Comment by ludvigak** [comment]
Hm, so the build process must have failed somehow.
You can try and see what happens if you build manually by running `build FINUFFT` from the Pkg REPL.
There should also be useful information in `deps/build.log`

**Comment by Kathryn-Drake** [comment]
Thank you for your response. I was able to fix a few issues using the errors from running build FINUFFT.
Now, the package has no errors on build, and the deps.jl file has been created. However, when I try to test the package, I get an error that the libfinufft.dylib is unable to load. When I run Libdl.dlopen_e("path_to_libfinufft.dylib"), it shows there is no pointer. I went into the library directory, and while the library file exists, the symbolic link pointing to it does not exist.
I'm assuming this means it didn't build properly. Any ideas?

**Comment by ludvigak** [comment]
First of all, what errors were you able to fix? I don't have a Mac myself, so it's useful to know what can go wrong and how to fix it.

Second, can you post the contents of your `build.log` file? That should hold some clues

**Comment by Kathryn-Drake** [comment]
First, I modified make.inc.macosx_clang to be the make.inc file, which resulted in no build errors. I'm still not sure why it couldn't find the libfinufft file. I have no experience with library files, so I wasn't able to fix the issue.
However, my advisor was able to successfully add and test the FINUFFT package on Julia v1.1. I had been using JuliaPro 1.1.1.1, but once I switched to Julia v1.1, I was able to add and test the package with no issues. I don't know what difference that makes, but it worked for me.
